### PR TITLE
add u.js

### PR DESCRIPTION
**Pull request by iamso**
added u.js, a micro framework / utility library, inspired by ki.js, youmightnotneedjquery.com and 140byt.es.

### CODE REVIEW

**Comment by jimaek** [comment]
This project seems too fresh and lacks of documentation. Can you please add it and ping me to check the PR again?

**Comment by iamso** [comment]
Documentation is in progress. I'll let you know when it's ready.

**Comment by iamso** [comment]
@jimaek doc is now available here https://github.com/iamso/u.js/blob/master/docs/api.md

File: files/u.js/update.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "packageManager": "github",
+  "name": "u.js",
+  "repo": "iamso/u.js",
+  "files": {
+  	"include": [
```
Comment (jimaek) [inline files/u.js/update.json]: instead of multiple includes just add `basePath:dist/` and then `include:*.min.*`

Comment (iamso) [inline files/u.js/update.json]: Could I just include everything from the dist folder? (incl. sourcemaps)

Comment (jimaek) [inline files/u.js/update.json]: *.min.* will include the maps as well. I just want to skip the unminified files

**Comment by iamso** [comment]
@jimaek can you check again please? hope this time I got it right ;)

File: files/u.js/update.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "packageManager": "github",
+  "name": "u.js",
+  "repo": "iamso/u.js",
+  "basePath": "dist/",
```
Comment (jimaek) [inline files/u.js/update.json]: basePath should be inside files. Check the examples https://github.com/jsdelivr/libgrabber#examples

**Comment by iamso** [comment]
@jimaek hope this doesn't exceed the limit of commits ;)

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge please

**Pull request closed without merging**